cms/views/cms_views.py

```diff
@@ -242,13 +295,47 @@ def content(self):
 
         if page.language != self.locale:
             raise HTTPNotFound()
-        return {
+
+        context = {
             'page': page,
             'linked_pages': linked_pages,
             'primary_category': category,
             'content': markdown(page.content),
             'description': markdown(page.description),
         }
+        context.update(self.get_comment_context(page))
+        return context
+
+    @view_config(route_name='comments',
+                 renderer='cms:templates/comments/comment_page.pt')
```

Comment:
That means existing sites will have to switch to Jinja templates if they want commenting. Is that okay?